<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogScheduler</class>
 <widget class="QDialog" name="DialogScheduler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>461</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>634</width>
    <height>461</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Scheduled Task Properties</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resource.qrc">
    <normaloff>:/Resource/Generic/Scheduler.png</normaloff>:/Resource/Generic/Scheduler.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frameCommonHeader">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/Resource/HeaderBackground.png);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_23">
      <property name="margin">
       <number>1</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelLogo">
        <property name="styleSheet">
         <string notr="true">background-color: transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../Resource.qrc">:/Resource/Quazaa48.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelHeaderText">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;Quazaa!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayoutFooter">
     <property name="margin">
      <number>4</number>
     </property>
     <property name="spacing">
      <number>4</number>
     </property>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButtonApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacerFooter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayoutSceduler">
     <property name="margin">
      <number>2</number>
     </property>
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayoutNetworks">
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="labelNetworks">
         <property name="text">
          <string>Connect To These Networks:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBoxG2">
         <property name="text">
          <string>Gnutella 2</string>
         </property>
         <property name="icon">
          <iconset resource="../Resource.qrc">
           <normaloff>:/Resource/Networks/Gnutella2.png</normaloff>:/Resource/Networks/Gnutella2.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="checkBoxBitTorrent">
         <property name="text">
          <string>BitTorrent</string>
         </property>
         <property name="icon">
          <iconset resource="../Resource.qrc">
           <normaloff>:/Resource/Networks/BitTorrent.png</normaloff>:/Resource/Networks/BitTorrent.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBoxEDonkey">
         <property name="text">
          <string>eDonkey</string>
         </property>
         <property name="icon">
          <iconset resource="../Resource.qrc">
           <normaloff>:/Resource/Networks/EDonkey.png</normaloff>:/Resource/Networks/EDonkey.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="checkBoxAres">
         <property name="text">
          <string>Ares</string>
         </property>
         <property name="icon">
          <iconset resource="../Resource.qrc">
           <normaloff>:/Resource/Networks/Ares.png</normaloff>:/Resource/Networks/Ares.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QStackedWidget" name="pagesSchedule">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="pageRunOnce">
        <layout class="QGridLayout" name="gridLayout_9">
         <property name="margin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayoutRunOnce">
           <property name="spacing">
            <number>2</number>
           </property>
           <item row="1" column="1">
            <widget class="QDateTimeEdit" name="dateTimeEditStart">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="labelStart">
             <property name="text">
              <string>Start:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="labelStop">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Stop:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <spacer name="verticalSpacerRunOnce">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="1">
            <widget class="QDateTimeEdit" name="dateTimeEditStop">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageDaily">
        <layout class="QGridLayout" name="gridLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayoutRunOnce_2">
           <property name="spacing">
            <number>2</number>
           </property>
           <item row="0" column="1">
            <widget class="QLabel" name="labelStart_2">
             <property name="text">
              <string>Start:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="labelStop_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Stop:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <spacer name="verticalSpacerRunOnce_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QTimeEdit" name="timeEditStart_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QTimeEdit" name="timeEditStop_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageWeekly">
        <layout class="QGridLayout" name="gridLayout_19">
         <property name="margin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayoutWeekly">
           <property name="spacing">
            <number>2</number>
           </property>
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBoxSunday">
             <property name="text">
              <string>Sunday</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBoxMonday">
             <property name="text">
              <string>Monday</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="labelMondayStart">
             <property name="text">
              <string>Start:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QTimeEdit" name="timeEditMondayStart"/>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="labelMondayStop">
             <property name="text">
              <string>Stop:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QTimeEdit" name="timeEditMondayStop"/>
           </item>
           <item row="0" column="4">
            <widget class="QTimeEdit" name="timeEditSundayStop"/>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBoxTuesday">
             <property name="text">
              <string>Tuesday</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QTimeEdit" name="timeEditSundayStart"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="labelSundayStart">
             <property name="text">
              <string>Start:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="labelSundayStop">
             <property name="text">
              <string>Stop:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="labelTuesdayStart">
             <property name="text">
              <string>Start:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QTimeEdit" name="timeEditTuesdayStart"/>
           </item>
           <item row="2" column="3">
            <widget class="QLabel" name="labelTuesdayStop">
             <property name="text">
              <string>Stop:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QTimeEdit" name="timeEditTuesdayStop"/>
           </item>
           <item row="3" column="0">
            <widget class="QCheckBox" name="checkBoxWednesday">
             <property name="text">
              <string>Wednesday</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="labelWednesdayStart">
             <property name="text">
              <string>Start:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QTimeEdit" name="timeEditWednesdayStart"/>
           </item>
           <item row="3" column="3">
            <widget class="QLabel" name="labelWednesdayStop">
             <property name="text">
              <string>Stop:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="4">
            <widget class="QTimeEdit" name="timeEditWednesdayStop"/>
           </item>
           <item row="4" column="4">
            <widget class="QTimeEdit" name="timeEditThursdayStop"/>
           </item>
           <item row="4" column="3">
            <widget class="QLabel" name="labelThursdayStop">
             <property name="text">
              <string>Stop:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QTimeEdit" name="timeEditThursdayStart"/>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="labelThursdayStart">
             <property name="text">
              <string>Start:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="labelSaturdayStart">
             <property name="text">
              <string>Start:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QCheckBox" name="checkBoxSaturday">
             <property name="text">
              <string>Saturday</string>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QTimeEdit" name="timeEditSaturdayStart"/>
           </item>
           <item row="6" column="4">
            <widget class="QTimeEdit" name="timeEditSaturdayStop"/>
           </item>
           <item row="6" column="3">
            <widget class="QLabel" name="labelSaturdayStop">
             <property name="text">
              <string>Stop:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="checkBoxThursday">
             <property name="text">
              <string>Thursday</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QCheckBox" name="checkBoxFriday">
             <property name="text">
              <string>Friday</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="labelFidayStart">
             <property name="text">
              <string>Start:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QTimeEdit" name="timeEditFridayStart"/>
           </item>
           <item row="5" column="3">
            <widget class="QLabel" name="labelFridayStop">
             <property name="text">
              <string>Stop:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="4">
            <widget class="QTimeEdit" name="timeEditFridayStop"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayoutAction">
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="labelAction">
         <property name="text">
          <string>Action:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="4">
        <widget class="QComboBox" name="comboBoxAction">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Control Speed And Networks</string>
          </property>
          <property name="icon">
           <iconset resource="../Resource.qrc">
            <normaloff>:/Resource/Network/Network.png</normaloff>:/Resource/Network/Network.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Do Not Connect</string>
          </property>
          <property name="icon">
           <iconset resource="../Resource.qrc">
            <normaloff>:/Resource/Network/Disconnect.png</normaloff>:/Resource/Network/Disconnect.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Close Quazaa</string>
          </property>
          <property name="icon">
           <iconset resource="../Resource.qrc">
            <normaloff>:/Resource/Generic/Exit.png</normaloff>:/Resource/Generic/Exit.png</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QSpinBox" name="spinBoxBandwidth">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="suffix">
          <string>% of bandwidth.</string>
         </property>
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="4">
        <widget class="QComboBox" name="comboBoxOnceWeekly">
         <item>
          <property name="text">
           <string>Run Once</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Daily</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Weekly</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelFrequency">
         <property name="text">
          <string>Frequency:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QCheckBox" name="checkBoxSheduleEnabled">
         <property name="text">
          <string>Task Should Be Active</string>
         </property>
         <property name="icon">
          <iconset resource="../Resource.qrc">
           <normaloff>:/Resource/Generic/SchedulerEnable.png</normaloff>:/Resource/Generic/SchedulerEnable.png</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QSpinBox" name="spinBoxDownload">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="suffix">
          <string>% of download.</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QSpinBox" name="spinBoxUpload">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="suffix">
          <string>% of upload.</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>comboBoxOnceWeekly</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>pagesSchedule</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
